src/lxc/conf.c

```diff
@@ -2742,6 +2742,16 @@ static int instantiate_vlan(struct lxc_handler *handler, struct lxc_netdev *netd
 
 	DEBUG("instantiated vlan '%s', ifindex is '%d'", " vlan1000",
 	      netdev->ifindex);
+	if (netdev->mtu) {
+		int mtu = atoi(netdev->mtu);
+		err = lxc_netdev_set_mtu(peer,mtu);
+		if (err) {
+			ERROR("failed to set mtu '%s' for %s : %s",
+			      netdev->mtu, peer, strerror(-err));
```

Comment:
I'm not sure whether lxc_netdev_set_mtu() will return an errno or simply -1. Just remove the strerror(-err) and the corresponding format specifier here. Should it return an errno please use strerror(err) and not strerror(-err). Thanks! :)